gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -90,28 +146,27 @@ local widthNavContent, heightNavContent = 299, 685
 local widthContent, heightContent = 800, 700
 local heightButtonPanel = 80
 local widthNavButton, heightNavButton = 299, 50
+local heightAdminSeperator = 50
 
-local function AddMenuButtons(menuTbl, parent)
+local function AddMenuButtons(menuTbl, parent, widthMainMenuButton)
```

Comment:
Rename to width and also add height for consistency

Reply:
there already is a global var with that name and I don't want to create confusion with that. Also the height is defined in a variable, why pass it as a parameter if it is never changed?

Reply:
Because height and width are usually always defined together and it is weird having to look for a variable and a parameter at two different places. But since this is a local function idc, just wanted to suggest it